app/js/app.js

```diff
@@ -251,10 +251,10 @@ KMCModule.config(['$routeProvider', '$locationProvider', '$httpProvider', '$tool
         }
     });
 	var kmc = window.parent.kmc;
-    if (kmc && kmc.vars.studio.showFlashStudio === false){
+    if (kmc && kmc.vars.studioV3.showFlashStudio === false){
```

Comment:
@yairans - what about a situation where the user have permissions to v2 and v3 studios but not v1? The subNav will be hidden and he will not be able to navigate between v2 and v3 studios.